src/gui/editors/PianoRoll.cpp

```diff
@@ -4590,6 +4590,93 @@ int PianoRoll::quantization() const
 	return DefaultTicksPerBar / Quantizations[m_quantizeModel.value() - 1];
 }
 
+int returnStrum(int st)//input > read value from comboBox Q. Output size of strum
```

Comment:
From what I can tell this just reimplements `PianoRoll::quantization()`

Reply:
If the method `PianoRoll::quantization()` can be used to add _each_ of the values in the Q-dropdown to a position-value, then we should ofcause use that method in my strumming-method. I cant see how  `PianoRoll::quantization()` would do that, but i will look at it again.
One question :
What is the procedure for making an edit, commit & PR that new edit.
Do I
* Edit the _current PR_ (I am not sure how to do that)
Or..
* Do I _close_ the current PR, and make a _new_ one with the _new_ and edited code?

Reply:
To update your PR, all you have to do is make changes to the source branch. So in this case, make any changes you want on the musikBear/master branch and they'll show up here.

Reply:
Woooo _That_ is actually clever!
A 👍 to gitHub for implementing that!
and i looked at `PianoRoll::quantization()` and ofcause you are correct. That method does almost the same as my switch. Its only `default case` where user choice is _"note-length"_ that isent directly covered. I will look at that.